src/annotator/config/config.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
This file isn't actually completely new, the diff is just messed up because I both moved and edited it. (Looking at the individual commits will help.)